Update vignette issues (#184)
mrc-ide/dust: vignettes/dust.Rmd

model$run(100)
```

> We use as many random number generators as their are particles, so if you run fewer particles or more, increase the threads or decrease, the results will be the same (see `vignette("design")` for more on this).

Reviewer: ```suggestion
We use as many random number generators as there are particles, so if you run fewer particles or more, increase the threads or decrease, the results will be the same (see `vignette("design")` for more on this).
```

cc_output(readLines(dust:::dust_file("examples/sir.cpp")))
```

> There is a bit different here to the version that is shown before for the walk model, but the core parts are only slightly different:

Reviewer: ```suggestion
There a few changes here compared to the version that was shown before for the walk model, but the core parts are only slightly different:
```
